app/kubernetes/views/resource-pools/edit/resourcePoolController.js

```diff
@@ -312,31 +275,30 @@ class KubernetesResourcePoolController {
         eventWarningCount: 0,
         canUseIngress: endpoint.Kubernetes.Configuration.IngressClasses.length,
         duplicates: {
-          ingressHosts: new KubernetesFormValueDuplicate(),
+          ingressHosts: new KubernetesFormValidationReferences(),
         },
       };
 
       this.state.activeTab = this.LocalStorage.getActiveTab('resourcePool');
 
       const name = this.$transition$.params().id;
 
-      const [nodes, pool] = await Promise.all([this.KubernetesNodeService.get(), this.KubernetesResourcePoolService.get(name)]);
+      const [nodes, pools] = await Promise.all([this.KubernetesNodeService.get(), this.KubernetesResourcePoolService.get()]);
 
-      this.pool = pool;
+      this.pool = _.find(pools, { Namespace: { Name: name } });
+      this.formValues = new KubernetesResourcePoolFormValues(KubernetesResourceQuotaDefaults);
+      this.formValues.Name = this.pool.Namespace.Name;
 
       _.forEach(nodes, (item) => {
         this.state.sliderMaxMemory += filesizeParser(item.Memory);
         this.state.sliderMaxCpu += item.CPU;
       });
       this.state.sliderMaxMemory = KubernetesResourceReservationHelper.megaBytesValue(this.state.sliderMaxMemory);
 
-      const quota = pool.Quota;
+      const quota = this.pool.Quota;
       if (quota) {
         this.oldQuota = angular.copy(quota);
-        this.formValues.HasQuota = true;
-        this.formValues.CpuLimit = quota.CpuLimit;
-        this.formValues.MemoryLimit = KubernetesResourceReservationHelper.megaBytesValue(quota.MemoryLimit);
-
+        this.formValues = KubernetesResourceQuotaConverter.quotaToResourcePoolFormValues(quota);
```

Comment:
Missing
`this.savedFormValues = angular.copy(this.formValues);`
at line 318/319 (line above the `catch(err)` )